packages/react/src/patterns/sub-patterns/Layout/__stories__/Layout.stories.js

```diff
@@ -78,76 +76,100 @@ storiesOf('Patterns (Sub-Patterns)|Layout', module)
-              quisque non tellus orci ac. At erat pellentesque adipiscing
-              commodo elit at. Amet tellus cras adipiscing enim eu turpis
-              egestas. Purus gravida quis blandit turpis cursus in hac
-              habitasse. Interdum velit laoreet id donec ultrices tincidunt arcu
-              non sodales. Interdum posuere lorem ipsum dolor sit. Quam quisque
-              id diam vel quam elementum pulvinar etiam non. Massa enim nec dui
-              nunc mattis enim. Ac tortor vitae purus faucibus ornare
-              suspendisse sed. Non pulvinar neque laoreet suspendisse.
-            </p>
-          </div>
+              <p style={{ paddingBottom: '1rem' }}>
+                Hac habitasse platea dictumst quisque sagittis. Faucibus nisl
+                tincidunt eget nullam non nisi est sit. Volutpat ac tincidunt
+                vitae semper quis lectus. Nisi lacus sed viverra tellus in hac.
+                Rhoncus dolor purus non enim. Suspendisse sed nisi lacus sed
+                viverra. Faucibus vitae aliquet nec ullamcorper sit amet risus
+                nullam eget. Augue interdum velit euismod in pellentesque massa.
+                Fermentum iaculis eu non diam phasellus vestibulum lorem sed.
+                Volutpat maecenas volutpat blandit aliquam etiam erat velit
+                scelerisque.
+              </p>
+
+              <p style={{ paddingBottom: '1rem' }}>
+                Mi in nulla posuere sollicitudin aliquam ultrices sagittis. Elit
+                pellentesque habitant morbi tristique senectus. Nunc non blandit
+                massa enim nec dui nunc mattis enim. In egestas erat imperdiet
+                sed euismod nisi porta lorem mollis. Donec pretium vulputate
+                sapien nec sagittis aliquam malesuada bibendum arcu. Auctor urna
+                nunc id cursus metus aliquam eleifend mi. Velit aliquet sagittis
+                id consectetur. Arcu cursus vitae congue mauris. Gravida rutrum
+                quisque non tellus orci ac. At erat pellentesque adipiscing
+                commodo elit at. Amet tellus cras adipiscing enim eu turpis
+                egestas. Purus gravida quis blandit turpis cursus in hac
+                habitasse. Interdum velit laoreet id donec ultrices tincidunt
+                arcu non sodales. Interdum posuere lorem ipsum dolor sit. Quam
+                quisque id diam vel quam elementum pulvinar etiam non. Massa
+                enim nec dui nunc mattis enim. Ac tortor vitae purus faucibus
+                ornare suspendisse sed. Non pulvinar neque laoreet suspendisse.
+              </p>
+            </div>
+            <div
+              data-sticky={select(
+                'Sticky right column',
+                ['true', 'false'],
+                'true'
+              )}
+              style={{
+                backgroundColor: 'white',
+              }}>
```

Comment:
Why are we setting the background color here?

Reply:
I think because in mobile the content starts to overlap and looks a little weird. This is more on the story level.